README.md

```diff
@@ -3,15 +3,14 @@
 ![Nuget](https://img.shields.io/nuget/v/Quantoz.Nexus.Sdk.Token)
 
 Welcome to the Quantoz Nexus SDK repository. In here you will find all the tools required to make your connection with Nexus even easier!
-
 ## Repository Overview
 
 This repository consists of the following projects:
 
-1. [Nexus Token SDK](./Nexus.Token.SDK) - This project contains functionalities required to interact with your Nexus Token environments.
```

Comment:
Also write about Nexus.Sdk.Shared

Reply:
I have no idea what to write about Nexus Sdk Shared :) i just updated the existing links which were 404 :)